portfolio/src/main/webapp/script.js

```diff
@@ -81,12 +81,18 @@ function createCommentElement(comment) {
   commentElement.classname = 'comment';
 
   const headerElement = document.createElement('span');
-  headerElement.innerHTML = "<h3>"+comment.posterName +"<h3>";
+  headerElement.innerHTML = "<h3>"+cleanseString(comment.posterName) +"<h3>";
 
   const bodyElement = document.createElement('span');
-  bodyElement.innerHTML = "<p>" + comment.comment +"<p>";
+  bodyElement.innerHTML = "<p>" + cleanseString(comment.comment) +"<p>";
 
   commentElement.appendChild(headerElement);
   commentElement.appendChild(bodyElement);
   return commentElement;
-}
\ No newline at end of file
+}
+// attempt to prevent html injection
+function cleanseString(html) {
+  // Thanks, https://stackoverflow.com/questions/20855482/preventing-html-and-script-injections-in-javascript
```

Comment:
Instead of having a comment like this you could explain what the code is trying to do, and add the link to the description to say this is the reference so people don't need to go to the link to understand why you are writing this.

Reply:
+1. Please clean up the comments to this function. Explain what you are trying to do, use capitalization and keep whitespace formatting. Write these comments as if other engineers are looking at your code

Reply:
This does not look resolved.

Reply:
@warfa-google do you mean that the changes aren't showing up on your end, or that I should be more thorough with the comment?